### ISSUE TITLE
Can't find information while using "Find in files"

### ISSUE THREAD

**Issue by hoangbachvan** [godotengine/godot#66697]
### Godot version

3.5

### System information

Window 11

### Issue description

I can't find or the result limited the word matching when using Find in Files function.

### Steps to reproduce

use Ctrl + Shift + F -> Fill the word -> find -> nothing found or limited appear

**Comment by MewPurPur** [comment]
Could you provide more details, maybe examples?

**Comment by nongvantinh** [comment]
We actually need more information here.

**Comment by Calinou** [comment]
Find in Files only searches in saved external scripts. It does not search in built-in scripts or in unsaved changes within the script editor.

**Comment by hoangbachvan** [comment]
Yes, my scene uses built-in to reduce a large number of files. Is this improvable?

**Comment by nongvantinh** [comment]
Step to reproduce the bug:

1. Create a new node, any node.
2. Add a script to that node.
3. Insert a paragraph into that script and do not save it.
4. Ctr + Shift + F or Search -> Find in files, find anything contain words in the paragraph above.
5. Result is nothing found.
7. Ctr + Left-Click on the `base` class's the script `extends` from to go the build-in document of that class.
8. Ctr + Shift + F or Search -> Find in files.
9. Fill in the words and click `Search`.
10. Nothing found from that search.

While current 4.0 `generate vsproj` option is broken. I'll take some time to analyse the error.

**Comment by YouveBeen** [comment]
I have the same issue using 4.0.1.
I was using find in files without troubles in 3.5 but since i use 4+ the search results is always empty **when** searching from the root folder.  (Folder settings is just "res://")

**Comment by yhwhey** [comment]
Is there any plan to support find in files searching built in scripts. Intuitively it should search them

**Comment by AThousandShips** [comment]
This is supported in 4.0.2 and later, not sure of any plans for 3.x

**Comment by yhwhey** [comment]
definitely doesn't work in 4.0.3 at the moment

**Comment by AThousandShips** [comment]
My bad this change was only for the script editor, not "Find In Files"

**Comment by TheDying0fLight** [comment]
Can this be closed? It does not seem to contain an unsolved/non-tracked issue

### Potential causes
- Built-in scripts: Now supported as mentioned by @AThousandShips
- Unsaved changes: Now supported outside of built-in scripts https://github.com/godotengine/godot/pull/114300
- .gdignore file in root: Tracked here https://github.com/godotengine/godot/issues/98116

**Comment by KoBeWi** [comment]
#110388 moved Find in Files extensions to Editor Settings, so you can now enable tscn search globally. Adding tscn to extension list is also a documented way to search built-in scripts, so this can be closed I think.
